Automattic/newspack-plugin
# Update the Welcome screen/component

### All Submissions:






### Changes proposed in this Pull Request:

_Note: works better alongside #316_

* Add specific class to body when Welcome Component is active
* This allows to have some specific CSS just for this screen
* Changed the Wizard Pagination so it returns an empty Fragment when viewing the Welcome Screen
* Added a couple of temporary fixes for the buttons

__Before:__
![1 before](https://user-images.githubusercontent.com/177929/69550561-1d099d80-0f93-11ea-983e-d1748a5dfc2e.png)

__After:__

![2.after](https://user-images.githubusercontent.com/177929/69550490-ffd4cf00-0f92-11ea-8111-bb9e9b7c5467.png)

### How to test the changes in this Pull Request:

1. Switch to this branch.
2. Start with the setup.
3. The first screen (Welcome) just be blue and it shouldn't display the steps/back button
4. Click "Get started". The background should be grey and you should see the steps/back button

### Other information:
